crates/prelude/src/lib.rs

```diff
@@ -61,14 +61,14 @@ cfg_if! {
         /// Collection types.
         pub mod collections {
             pub use self::{
+                BinaryHeap,
                 BTreeMap,
                 BTreeSet,
-                BinaryHeap,
                 LinkedList,
                 VecDeque,
             };
-            pub use alloc::collections::*;
             pub use core::ops::Bound;
+            pub use alloc::collections::*;
```

Comment:
how does this ordering improve the code? before that it was alphabetically sorted

Reply:
if and else ordering 
They should be consistent
This is a good way to check the code and not easy to make mistakes